history-of-quakers/original/12-chapter-10.adoc

```diff
@@ -591,7 +652,8 @@ and reasonable words to them though not in the eloquent language of man`'s wisdo
 yet it is in the truth which is honorable.
 We are friends to righteousness and truth, and to all that are found therein.
 
-ROBERT GRASSINGHAM, GEORGE FOX.
+[.signed-section-signature]
+Robert Grassingham, George Fox.
```

Comment:
the other missing `--` end should go below here.

Reply:
That's so weird I didn't catch those because I've been comparing the original and modernized PDF side-by-side before I send it so if I did something like that in one and not the other, it would be easy to catch. Ugh. :)